src/win32/posix_w32.c

```diff
@@ -145,7 +137,7 @@ static bool path_is_volume(wchar_t *target, size_t target_len)
 
 /* On success, returns the length, in characters, of the path stored in dest.
  * On failure, returns a negative value. */
-static int readlink_w(
+int readlink_w(
```

Comment:
I'll add a new posix_w32.h file then for Win32 specific method that fulfill or enable a posix like function.